app/src/main/java/com/example/paetz/yacguide/RouteActivity.java

```diff
@@ -50,6 +51,15 @@ public void back(View v) {
         finish();
     }
 
+    public void showMap(View v) {
+        Uri gmmIntentUri = Uri.parse("geo:"+ _rock.getLatitude() + "," + _rock.getLongitude());
+        Intent mapIntent = new Intent(Intent.ACTION_VIEW, gmmIntentUri);
+        if (mapIntent.resolveActivity(getPackageManager()) != null) {
+            startActivity(mapIntent);
```

Comment:
I tested with google maps on the emulator and the coords were NOT represented with a marker. Can we somehow make sure that the map tool provides a proper marker for the given coords?

Reply:
Good question, we could request a whitelisted set of map tools...

Reply:
I investigated a bit more... There is absolutely now way to tell the Uri that there should be a marker. We can either query different apps one by one, leave it as is or integrate a map toolkit (which I considered too much work right now)

Reply:
Ok, then we leave it as is for now, since it's better than nothing